scripts/2015_nips_paper/run/run_without_metalearning.py

```diff
@@ -0,0 +1,86 @@
+import argparse
```

Comment:
This file is almost equal to the one above. Could you please combine them into a single file to keep the amount of code to reproduce the experiments shorter?

Reply:
I agree that they can be merged into one file. In fact, I would merge run_with_metalearning, run_without_metalearning, and run_auto_sklearn into one file as most of what run_auto_sklearn does is just calling the other two files.